src/compiler/emitter.ts

```diff
@@ -379,6 +379,93 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
             return true;
         }
 
+        const enum Jump {

```

Comment:
Actually `Jump` is fine as long as you just document what these are for (i.e. tracking non-local jumps for local variable capturing in loops)